packages/next-tinacms-github/src/get-github-static-props.ts

```diff
@@ -66,6 +66,14 @@ export async function getGithubStaticProps(
     )
   } catch (e) {
     if (e instanceof GithubError) {
+      console.error(
+        githubErrorMessage({
+          path: options.fileRelativePath,
+          sourceProvider,
+          accessToken,
```

Comment:
I probably wouldn't want the user's access token logged

Reply:
Legit. I want to know it's there but shouldn't show the whole thing

Reply:
adjusted